BUILD.md

```diff
@@ -9,103 +9,136 @@ See also README.md Dependencies section
 
 Checkout the .travis.yml for build bootstrap without root/sudo requirement
 
-Notes:
+# Notes:
  On Ubuntu 14.04 - to get g++ 4.9
+ ```
  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
  sudo apt-get upgrade
+ ```
 If using a vmware image/starting fresh
+```
  sudo vmware-config-tools.pl  # to setup shared folders
-
-# Cmake 3.2 or later
-
-See below for mac (Cmake 3.3/head on a mac is recommended for Xcode support)
-
+```
+# Build Instructions
+__Install Cmake 3.2 or greater.__
+*See directions below for Mac
+(Cmake 3.3/head on a Mac is recommended for Xcode support)*
+```
 wget http://www.cmake.org/files/v3.2/cmake-3.2.3.tar.gz
 tar xvfz cmake-3.2.3.tar.gz
 cd cmake-3.2.3
 ./bootstrap --prefix=/usr --parallel=16 && make -j && sudo make install
-
-# Get folly
+```
+__Get folly source tree__
+```
+mkdir deps/; cd deps
 git clone https://github.com/facebook/folly.git
+```
+__Install glog-dev (includes gflags, libunwind), boost system, double conversion
+if you can find a binary distrubution for your variant of linux:__
+*libjemalloc-dev dependency is optional*
 
-# Getting pre built dependencies: glog-dev (which adds gflags, libunwind)
-# boost system, double conversion
+```
 sudo apt-get install libgoogle-glog-dev libboost-system-dev \
-     libdouble-conversion-dev # and optionally: libjemalloc-dev
-
-# OR
-
-# If double-conversion isn't available:
-
+libdouble-conversion-dev libjemalloc-dev
+```
+*If double-conversion isn't available via apt-get:*
+```
 git clone https://github.com/floitsch/double-conversion.git
 cd double-conversion; cmake . ; make -j && sudo make install
+```
 
+__Otherwise, Build gflags and glog from source__
```

Comment:
let's make this
__Otherwise, Build double-conversion, gflags and glog from source__
and delete line 45 and move the double conversion text after this heading